docs/reference/commandline/swarm_init.md

```diff
@@ -130,6 +131,21 @@ management traffic of the cluster.
 If unspecified, Docker will use the same IP address or interface that is used for the
 advertise address.
 
+### `--data-path-port`
+
+This flag specifies data path port number to use for data path traffic (1024 - 49151).
```

Comment:
Looking at this again, perhaps it's better to leave out the `--data-path-addr` option, otherwise we'll have to explain here that `--data-path-addr` only applies to the node that you're connected with, and not for the whole cluster (wheras `--data-path-port` is a cluster-wide configuration).

I tried rewriting it a bit; let me know if you like this suggestion (**note** I had to indent the suggestion below, due to the nested ` ``` ` );

```suggestion
	This flag allows you to configure the UDP port number to use for data path
	traffic. The provided port nummer must be within the 1024 - 49151 range. If
	this flag is not set or is set to 0, the default port (4789) is used.

	The data path port can only be configured when initializing the swarm, and
	applies to all nodes that join the swarm.

	The following example initializes a new Swarm, and configures the data path
	port to UDP port 7777;

	```bash
	docker swarm init --data-path-addr=20.20.20.1 --data-path-port=7777
	```

	After the swarm is initialized, use the `docker info` command to verify that
	the port is configured:

	```bash
	docker info

	...
	ClusterID: 9vs5ygs0gguyyec4iqf2314c0
	Managers: 1
	Nodes: 1
	Data Path Port: 7777
	...
	```
```

Reply:
did small corrections and update it. Pls check it out.